openwisp_controller/config/api/views.py

```
@@ -289,6 +303,60 @@ def certificate_delete_invalidates_cache(cls, organization_id, common_name):
+
+class RevisionRestoreView(ProtectedAPIMixin, GenericAPIView):
+    serializer_class = serializers.Serializer
+    queryset = Version.objects.select_related("revision").order_by(
+        "-revision__date_created"
+    )
+
+    def get_queryset(self):
+        model = self.kwargs.get("model").lower()
+        return self.queryset.filter(content_type__model=model)
+
+    def post(self, request, *args, **kwargs):
```

Ruff (0.14.10)

Unused method argument: `args`

(ARG002)